modules/trustpidSystem.md

```diff
@@ -38,7 +15,7 @@ pbjs.setConfig({
 | name | Required | String | The name of the module | `"trustpid"`
 | params | Required | Object | Object with configuration parameters for trustpid User Id submodule | - |
 | params.maxDelayTime | Required | Integer | Max amount of time (in seconds) before looking into storage for data | 2500 |
-| bidders | Required | Array of Strings | An array of bidder codes to which this user ID may be sent. Currently required and supporting AdformOpenRTB | `["adf"]` |
+| bidders | Required | Array of Strings | An array of bidder codes to which this user ID may be sent. Currently required and supporting AdformOpenRTB | [`"adf"`, `"adformPBS"`, `"ix"`] |
```

Comment:
This array is required from TrustPID usage perspective. It is handled by our module with correct settings that we set up per entity.

Which rules violation do you have in mind?

I realised that the back ticks are supposed to be outside of the array instead of around every element. I will fix that.

Reply:
yes; also, it isnt required